config/generic-mightyboard.cfg

```diff
@@ -117,3 +117,16 @@ scl_pin: PJ5
 sda_pin: PJ6
 wiper: 0.50
 scale: 0.773
+
+[display]
+lcd_type: hd44780_spi
+spi_software_mosi_pin: PC3
+spi_software_sclk_pin: PC2
+#miso not used, dummy pin.
+spi_software_miso_pin: PJ1
+latch_pin: PC4
```

Comment:
In general, it looks fine, but it does look like a `latch_pin` config option was added and not documented.

-Kevin

Reply:
I added the documentation.